### PR TITLE
Use Stream instead of Span<byte> for deserializing

### PR DESCRIPTION
**Pull request by CPKreuz**

### CODE REVIEW

**Review by flabbet — COMMENTED** [review]
So basically, you reverse your changes? I mean you used byte[] before and changed to Span<byte>

**Comment by CPKreuz** [comment]
> So basically, you reverse your changes? I mean you used byte[] before and changed to Span

Well, first I used Span<byte> incorrectly, than I used it correctly but that introduced a bug, now I don't use it and it seems to work

**Review by flabbet — CHANGES_REQUESTED** [review]
Bug: Create a new layer without any pixels, save the file, open recent, select the file: crash.

Also bump package version

**Comment by flabbet** [comment]
Everything works now, but one of the solution files has syntax errors. Also, bump package version.
image

File: src/PixiParser/Helpers.cs [hunk 1]
```diff
@@ -1,19 +1,20 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Drawing;
```
Comment (flabbet) [inline src/PixiParser/Helpers.cs:3]: The parser should use primitives, such as a List of bytes. PixiEditor doesn't even use System.Drawing.Color, but System.Windows.Media.Color. But it's not cross-platform, so just use bytes instead

**Review by flabbet — CHANGES_REQUESTED** [review]

**Review by flabbet — APPROVED** [review]

**Pull request merged**